ntpclient/ntpclient_test.go

```diff
@@ -0,0 +1,27 @@
+package ntpclient
+
+import (
+	"testing"
+)
+
+func TestNTPClient(t *testing.T) {
+	pool := []string{"pool.ntp.org:123", "0.pool.ntp.org:123"}
+	NTPTime, err := NTPClient(pool)
+	if err != nil {
+		t.Errorf("failed to get time %v", err)
+	}
+	if NTPTime.IsZero() {
```

Comment:
This might be redundant due to being returned using as time.Unix() and doesn't switch IsZero properly. That function is looking for -62135596800.